Packages/Core/Runtime/Preferences/AtomsPreferences.cs

```diff
@@ -0,0 +1,99 @@
+﻿#if UNITY_EDITOR
```

Comment:
No. Since AtomPreferences contains properties that needs to be accessed by classes in the runtime assembly (eg. `IsDebugModeEnabled`) it also needs to reside there.